### ISSUE TITLE
Trying with `grep "--- FAIL:"` nothing happens

### ISSUE THREAD

**Issue by mrkam2** [ryu1kn/vscode-edit-with-shell#2]

**Comment by mrkam2** [comment]
grep '--- FAIL:' give the same result

grep FAIL works, but has more results than I need

**Comment by ryu1kn** [comment]
Hello @mrkam2 , I think it's because of the way you gave the pattern to `grep`. Can you try `grep -- "--- FAIL:"` ?

I get an error even on console if I do `grep '--- FAIL:'`

```sh
$ cat __grep.txt

- SUCCESS:
--- FAIL:
--- SUCCESS:
-- FAIL:

$ cat __grep.txt | grep "--- FAIL:"
grep: unrecognized option `--- FAIL:'
usage: grep [-abcDEFGHhIiJLlmnOoqRSsUVvwxZ] [-A num] [-B num] [-C[num]]
        [-e pattern] [-f file] [--binary-files=value] [--color=when]
        [--context[=num]] [--directories=action] [--label] [--line-buffered]
        [--null] [pattern] [file ...]

$ cat __grep.txt| grep -- "--- FAIL:"
--- FAIL:
```

**Comment by ryu1kn** [comment]
Having said that, the extension should give an error message if the command failed

**Comment by mrkam2** [comment]
Thanks! You're right, the command was wrong. But a debug message is always
helpful.

**Comment by ryu1kn** [comment]
Yeah, the error message box should appear on error, but it doesn’t. That’s something actually unexpected. I’ll check it.

**Comment by ryu1kn** [comment]
> the error message box should appear on error, but it doesn’t

Found out that it was caused by the character sequence `]]` in grep's usage output. I'm using [`vscode.window.showErrorMessage`](https://code.visualstudio.com/docs/extensionAPI/vscode-api#window.showInformationMessage) to display the information about failure, but this function interprets certain character sequences as markup.

`HtmlContentRenderer.renderFormattedText`

* [src/vs/base/browser/htmlContentRenderer.ts]( https://github.com/Microsoft/vscode/blob/21c8f52d0ca2bc44498f79dd304a0c2a3c5308e1/src/vs/base/browser/htmlContentRenderer.ts#L38)
* [src/vs/base/test/browser/htmlContent.test.ts](https://github.com/Microsoft/vscode/blob/3a13840209b431c39c0c4f14b6fad0ffe41c1456/src/vs/base/test/browser/htmlContent.test.ts)

We can prevent the text being recognised as markup-ed text by escaping with `\`.

**Comment by ryu1kn** [comment]
Released as v0.2.2 👍
